src/sorting/python/selection_sort/src/selection_sort.py

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Your code looks nice, it needs a module header to be added on top.
The other thing is that the testing file looks updated and that caused the following error when testing your code:
`ModuleNotFoundError: No module named 'src.sorting  ` you just need to use the original testing file and your code will pass the test just fine